<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>180</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pause">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_stop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_goto">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>GoTo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_goto"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_points">
      <attribute name="title">
       <string>Points</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_points">
         <item row="4" column="0">
          <widget class="QLabel" name="label_loop_from">
           <property name="text">
            <string>Loop from</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QSpinBox" name="spinBox_loop_from"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_loop_to">
           <property name="text">
            <string>Loop to</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QSpinBox" name="spinBox_loop_to"/>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="checkBox_loop_fromlast">
           <property name="text">
            <string>from last</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="checkBox_loop_tofirst">
           <property name="text">
            <string>to first</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_loop_start">
           <property name="text">
            <string>Start from</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_point_first">
           <property name="text">
            <string>from first</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="spinBox_point_first"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_loop_do">
           <property name="text">
            <string>Do Loop</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_point_last">
           <property name="text">
            <string>last</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinBox_point_last"/>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="spinBox_loop_times"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_loop_end">
           <property name="text">
            <string>End@</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_speed">
      <attribute name="title">
       <string>Speed</string>
      </attribute>
      <widget class="QRadioButton" name="radioButton_S3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>time</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_s1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>131</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>speed from time</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_S2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>time from speed</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_trg">
      <attribute name="title">
       <string>Trigger</string>
      </attribute>
      <widget class="QRadioButton" name="radioButton_t2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next on Trigger</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_t1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next on Arrival</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_opt">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_loop_do_2">
         <property name="text">
          <string>Highlight last point in Editor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_timen">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time next </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_time_next">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_timea">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time akt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_time_akt">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_Values">
     <item row="0" column="0">
      <widget class="QLabel" name="label_akt">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Akt. Pos.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_pos_akt">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_next">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next Pos.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_pos_next">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
